src/components/aktivitetskrav/vurdering/VurderAktivitetskravSkjema.tsx

```diff
@@ -0,0 +1,63 @@
+import { FlexRow, PaddingSize } from "@/components/Layout";
+import React, { ReactElement } from "react";
+import { CreateAktivitetskravVurderingDTO } from "@/data/aktivitetskrav/aktivitetskravTypes";
+import { Innholdstittel } from "nav-frontend-typografi";
+import { SkjemaInnsendingFeil } from "@/components/SkjemaInnsendingFeil";
+import { VurderAktivitetskravSkjemaButtons } from "@/components/aktivitetskrav/vurdering/VurderAktivitetskravSkjemaButtons";
+import { Form } from "react-final-form";
+import { useVurderAktivitetskrav } from "@/data/aktivitetskrav/useVurderAktivitetskrav";
+import { VurderAktivitetskravBeskrivelse } from "@/components/aktivitetskrav/vurdering/VurderAktivitetskravBeskrivelse";
+import { ValidationErrors } from "final-form";
+
+interface VurderAktivitetskravSkjemaProps<SkjemaValues> {
+  title: string;
+  arsakVelger: ReactElement;
+  setModalOpen: (modalOpen: boolean) => void;
+  aktivitetskravUuid: string;
+
+  toDto(values: SkjemaValues): CreateAktivitetskravVurderingDTO;
+
+  validate(values: Partial<SkjemaValues>): ValidationErrors;
+}
+
+export const VurderAktivitetskravSkjema = <SkjemaValues extends object>({
```

Comment:
Hmm, føler dette er en lur måte å løse det på! Har vi gjort sånn her andre steder i frontendene våre og?

Reply:
Ikke som jeg kommer på, men det kan godt hende. Fint med litt typescript-generics bare man ikke drar det for langt 😄